rosmip_joint_publisher:
  type: joint_state_controller/JointStateController
  publish_rate: 50

rosmip_imu_controller:
  type: imu_sensor_controller/ImuSensorController
  publish_rate: 50

rosmip_balance_controller:
    type: rosmip_control/RosMipControllerLegacy
    odom_ws: 0.0823
    odom_lr: 0.04
    odom_rr: 0.04
    pose_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
    twist_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
    
    #    enable_odom_tf: false
